src/Tribe/Settings_Tab.php

```diff
@@ -349,6 +366,147 @@ public function get_priority(): string {
 		return $this->priority;
 	}
 
+	/**
+	 * Determine the field value.
+	 *
+	 * @since TBD
+	 *
+	 * @param array  $field The field array.
+	 * @param string $key   The field key.
+	 *
+	 * @return mixed The field value.
+	 */
+	protected function get_field_value( $field, $key ) {
+		// Some options should always be stored at network level.
+		$network_option = (bool) ( $field['network'] ?? false );
+
+		if ( is_network_admin() ) {
+			$parent_option = $field['parent_option'] ?? Tribe__Main::OPTIONNAMENETWORK;
+		} else {
+			$parent_option = $field['parent_option'] ?? Tribe__Main::OPTIONNAME;
+		}
+
+		/**
+		 * Get the field's parent_option in order to later get the field's value.
+		 *
+		 * @param string $parent_option The parent option name.
+		 * @param string $key           The field key.
+		 */
+		$parent_option = apply_filters( 'tribe_settings_do_content_parent_option', $parent_option, $key );
```

Comment:
Any _new_ filters should be namespaced properly: https://docs.theeventscalendar.com/developer/plugins/namespacing/#prefixes

So this should really be `tec_settings_do_content_parent_option`

Reply:
This isn't actually a new filter, it was just relocated, and I added a docblock that was missing previously.